src/backend/services/carValueService.ts

```diff
@@ -0,0 +1,43 @@
+interface CarValue {
```

Comment:
interface carValue could be changed to something like carInfo or carDetails (more descriptive of what it is) as carValue is used again as a variable in your calculateCarValue function, can be a little confusing having the same words used for different things if that makes sense

Reply:
makes sense thanks i will change it to carDetails

Reply:
great idea! i will try make changes before mission 4